content/accessibility/announcements.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Announcements
```

Comment:
```suggestion
title: Assistive Technology Announcements
```

Reply:
I updated the suggestion to follow our style guide (sentence case titles)

Reply:
Thanks @inkblotty and @yaili! I updated the page title in https://github.com/primer/design/pull/237/commits/abef191552d03c4119124f30df944d96d22068ac and the title in the nav in https://github.com/primer/design/pull/237/commits/342dc04cd5aabda26d59296713357107b60bb14c